src/services/db/RepoService.ts

```diff
@@ -112,19 +116,39 @@ export default class RepoService extends GitHubService {
     return ReviewApi.createComment(siteName, pullRequestNumber, user, message)
   }
 
-  getFilePath({ siteName, fileName, directoryName }: any): any {
+  getFilePath({
+    siteName,
+    fileName,
+    directoryName,
+  }: {
+    siteName: string
+    fileName: string
+    directoryName?: string
+  }): string {
     return super.getFilePath({
       siteName,
       fileName,
       directoryName,
     })
   }
 
-  getBlobPath({ siteName, fileSha }: any): any {
+  getBlobPath({
+    siteName,
+    fileSha,
+  }: {
+    siteName: string
+    fileSha: string
+  }): string {
```

Comment:
nit: This `string` return type can be inferred from the parent so we don't need to redefine here

Reply:
Sure, thanks for the catch. Removed in ce3615eb571009f5f1bf3074191def1847dfa87b